### PR TITLE
Removing Avenir from webfonts

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Removing Avenir from our default webfonts as part of the road to 1.0
## Additions
- Added webfont variables to make it easy to override defaults in cf-theme-overrides.less
## Removals
- Removed licensed-fonts.less
- Removed references to Avenir
## Changes
- Ran `grunt vendor` to update dependencies
## Review
- @Scotchester 
## Preview

[Preview this PR without the whitespace changes](?w=0)
## Notes
- Part of https://github.com/cfpb/capital-framework/issues/161
- Related PR: https://github.com/cfpb/generator-cf/pull/69

### CODE REVIEW

**Comment by contolini** [comment]
Note: This is failing because I enabled Travis for this repo but the tests haven't been merged yet. Please ignore the :x:s above.

**Comment by ascott1** [comment]
Great change. This puts us closer to making CF more extensible for non-CFPB branded projects. :boom: :rocket:

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 
+## 0.11.0 - 2015-05-13
+
+### Removed
+- Removing CFPB fonts. To add your own custom fonts,
```
Comment (Scotchester) [inline CHANGELOG.md:10]: Might want to finish this sentence in a subsequent PR ;)

Comment (KimberlyMunoz) [inline CHANGELOG.md:10]: Yeah... It's in #38. Want to merge? ;)

Comment (Scotchester) [inline CHANGELOG.md:10]: So creepy that we were working on that at the same time...

**Pull request merged**